webapp/templates/partials/analysis/contest_stats.html.twig

```diff
@@ -0,0 +1,26 @@
+<div class="col-md-3 mt-3">
+    <div class="card">
+        <div class="card-header">
+            Contest Stats
+        </div>
+        <div class="card-body">
+            {% if show_misery|default(false) %}
+                <abbr data-toggle="popover" data-placement="bottom"
+                      title="Misery Index"
+                      data-content="
+                This is a measure of how 'bored' contestants are. It is the average number of
+                people-minutes spent fruitlessly trying to solve problems at the end of the contest.
+                <br>
+                Calculated as:<br>
+                average across all teams of [3 people x (number of minutes elapsed - minute last problem solved by team t)]
+              ">Misery
+                    index</abbr>: {{ misc.misery_index|number_format(0) }} minutes
+                <br>
+            {% endif %}
+            # Submissions: {{ misc.total_submissions }}<br>
+            # Accepted Submissions: {{ misc.total_accepted }} <span
+                class="text-muted">(~{{ (misc.total_submissions>0 ? (misc.total_accepted/misc.total_submissions)*100:0)|number_format(0) }}%)</span><br>
+            # of Teams: {{ misc.num_teams }}<br>
```

Comment:
The third bullet is the only one that has the word "or" in it. Is the "#" even needed or could it just say: "Submissions: 1353"?

Reply:
Note that this didn't change in this PR. But I can agree we can remove the # (and the 'of')